ui/src/app/applications/components/utils.tsx

```diff
@@ -374,17 +374,16 @@ export const MaintenanceWindowStatusIcon = ({state, window}: { state: appModels.
     return <React.Fragment><i title={current} className={className} style={{color}}/> {current}</React.Fragment>;
 };
 
-export const ApplicationMaintenanceWindowStatusIcon = ({state}: { state: appModels.ApplicationMaintenanceState }) => {
-    let className = 'fa fa-question-circle';
-    let color = COLORS.maintenance_state.unknown;
-    const name = 'Maintenance';
-
-    if (state.windows !== undefined && state.windows.length > 0) {
+export const ApplicationMaintenanceWindowStatusIcon = ({project, state}: { project: string, state: appModels.ApplicationMaintenanceState }) => {
+    let className = 'fa fa-stop-circle';
```

Comment:
i had trouble understanding the logic here, the `question-circle` only appears in an edge case, wonder if this is is bug or not

Reply:
I think that I left it there in case the state could not be looked up.